### PR TITLE
See More Questions Button Functionality

### PR DESCRIPTION
**Pull request by ygroener**
Added a See More Questions Button that increments length of question list by 2, and then disappears when there are no more questions to be displayed.
Added a collapse button to bring length of questions back to 2.

I also changed the sample data for questions back to the original length of 4. I had increased it to test functionality.

### CODE REVIEW

**Comment by ygroener** [comment]
https://user-images.githubusercontent.com/84756768/140584250-a43ce3e2-be1f-4a4d-8285-1f35a1396e03.mp4

**Comment by ygroener** [comment]
[Question List_ More Answered Questions button functionality](https://trello.com/c/XnXcwUuI/144-question-list-more-answered-questions-button-functionality)

File: src/components/questions/QuestionList.jsx [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
+    if (items.length > displayedQuestions.length) {
+      let length = displayedQuestions.length;
+      setDisplayed(items.slice(0, length + 2));
+    }
+  }
+
+  // This stores the height of the client screen
   const screenHeight = window.innerHeight;
+
+  // this is the hook for tracking the height of the questions list element
```
Comment (merosenlund) [inline src/components/questions/QuestionList.jsx:31]: I stumbled across another way to do this that might be simpler if you want to check it out at some point.

**Review by merosenlund — APPROVED** [review]
Looks good!

**Pull request merged**